src/rofi_rbw/rbw.py

```diff
@@ -17,7 +17,7 @@ def list_entries(self) -> List[Entry]:
             exit(2)
 
         return sorted(
-            [self.__parse_rbw_output(it) for it in (rbw.stdout.strip().split("\n"))],
+            [self.__parse_rbw_output(it) for it in (rbw.stdout[:-1].split("\n"))],
```

Comment:
I think I find `split('\n')` easier to understand. Would it be possible to change?

Reply:
It was. Thanks for the fast review. Locally it works.